control_panel_api/services.py

```diff
@@ -121,3 +121,15 @@ def delete_bucket_policies(name):
     policy_arn_readonly = _policy_arn(bucket_name, readwrite=False)
     aws.detach_policy_from_entities(policy_arn_readonly)
     aws.delete_policy(policy_arn_readonly)
+
+
+def bucket_create(name):
```

Comment:
One general observation about the names of these functions in this module (and probably in general).

I would rename these to make very explicit what's "private", e.g. internal utility function (at least for now) and what's the public interface of the module.

So for example we may have `services.create_bucket()` which calls internal things (e.g. `_create_bucket()` and `_create_bucket_policies()`).

I like the fact that someone can get in this module without having much knowledge of it and have a feeling of what can move around and what's a contract with the client code (which you can still change if needed, but you'll need to be more careful).